Neos.Flow/Classes/Command/ResourceCommandController.php

```diff
@@ -221,6 +221,7 @@ public function cleanCommand()
         $resourcesCount = $this->resourceRepository->countAll();
         $this->output->progressStart($resourcesCount);
 
+        /** @var list<PersistentResource> $brokenResources */
```

Comment:
what's that type hint, is `list` supported by common IDEs or should this be:

```suggestion
        /** @var array<PersistentResource> $brokenResources */
```

Reply:
yes it is well understood and its a short form of `array<int, mixed>`. I use it to denote further that the keys are numeric and (should) be simply ascending without holes. See `array_is_list`.

For example `array<int, string>` might mean that its a map of constant values to something else, of course the type is the same for phpstan, but visually its not.

Reply:
TIL, nice